gramps/gen/plug/utils.py

```diff
@@ -116,7 +116,7 @@ def extractall(self, path, members=None):
                 os.mkdir(fullname)
         for name in self.get_files(names):
             fullname = os.path.join(path, name)
-            outfile = file(fullname, 'wb')
+            outfile = open(fullname, 'wb')
             outfile.write(self.zip_obj.read(name))
             outfile.close()
```

Comment:
Even better to use a context here:
```suggestion
            with open(fullname, 'wb') as outfile:
                outfile.write(self.zip_obj.read(name))
```